README.md

```diff
@@ -1,36 +1,8 @@
-# Kaltura Player JS Platform - Cloud TV and OVP Media Players Based on the [PlayKit JS Player]
+# Massive Fork of Kaltura Player
```

Comment:
Please revert the readme and add an extra readme for the minimal version

Reply:
done